packages/preview-middleware-client/src/adp/quick-actions/fe-v2/lr-enable-semantic-date-range-filter-bar.ts

```diff
@@ -25,7 +26,19 @@ export class ToggleSemanticDateRangeFilterBar
     readonly forceRefreshAfterExecution = true;
     private isUseDateRangeTypeEnabled = false;
 
-    initialize(): void {
+    async initialize(): Promise<void> {
+        const version = await getUi5Version();
+        const isUI5VersionNotSupported =
+            isLowerThanMinimalUi5Version(version, { major: 1, minor: 128 }) &&
+            !(
+                isEqualToUi5Version(version, { major: 1, minor: 96, patch: 37 }, true) ||
```

Comment:
Higher patch versions should also be supported - e.g. it should also work for `1.96.38`, `1.96.39` etc.

Reply:
done.